Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "YAML"
 uuid = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
-version = "0.4.3"
+version = "0.4.4"
```

Comment:
Does it make sense to have version bumps occur in PRs? In my own repos, I tend to make separate commits where the only thing that happens is a version bump.

Doesn't make a huge difference to me, except that, in a case where multiple PRs are happening at the same time, there may be conflicts.

Reply:
I usually do it. It doesn't conflict if other PRs bump to the same version. But I can totally remove it if you want!